core-bundle/src/EventListener/CommandSchedulerListener.php

```diff
@@ -55,11 +55,7 @@ private function shouldRunCron(Request $request): bool
             return false;
         }
 
-        if ($this->autoMode && $this->cron->hasMinutelyCliCron()) {
-            return false;
-        }
-
-        return true;
+        return !($this->autoMode && $this->cron->hasMinutelyCliCron());
```

Comment:
@ausi Would `return !$this->autoMode || !$this->cron->hasMinutelyCliCron();` be the same?

Reply:
According to [De Morgan’s laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws), yes 🤓

Reply:
But the old version with `if ($this->autoMode && $this->cron->hasMinutelyCliCron()) {` is way more readable IMO.

Reply:
Reverted in f8fc72b51bd70224eb51a04d462fc2e3ce04206b.